btcwallet.go

```diff
@@ -24,27 +24,38 @@ var (
 	cfg *config
 )
 
-func main() {
+/*func main() {
```

Comment:
please clean code. If we will need to run the wallet as console app then `cmd/btcwallet/main.go` would be a right place for main method